src/khoj/processor/url/url_to_jsonl.py

```diff
@@ -0,0 +1,107 @@
+# Standard Packages
+import glob
+import logging
+from pathlib import Path
+from typing import List
+
+# External Packages
+from langchain.document_loaders import PyPDFLoader
+
+# Internal Packages
+from khoj.processor.text_to_jsonl import TextToJsonl
+from khoj.utils.helpers import get_absolute_path, is_none_or_empty, timer
+from khoj.utils.jsonl import compress_jsonl_data
+from khoj.utils.rawconfig import Entry, TextConfigBase
+from khoj.processor.url.web_crawler import scrape_text_with_selenium, get_webdriver
+
+logger = logging.getLogger(__name__)
+
+
+class URLToJsonl(TextToJsonl):
+    # Define Functions
+    def process(self, previous_entries=[]):
+        # Extract required fields from config
+        urls, output_file = (
+            self.config.input_files,
+            self.config.compressed_jsonl,
+        )
+
+        urls = [str(u) for u in urls]
+        logger.debug(f"input urls = {urls}")
+
+        # Input Validation
+        if is_none_or_empty(urls):
+            print("At least one of pdf-files or pdf-file-filter is required to be specified")
```

Comment:
Please use `logger` for any outputs going to the console. Here, you should use `logger.error`.